src/plugins/intel_cpu/src/emitters/snippets/aarch64/kernel_executors/gemm_copy_b.hpp

```diff
@@ -76,6 +79,7 @@ struct GemmCopyBKernelKaiConfig : public snippets::KernelExecutorBase::GenericCo
     std::shared_ptr<StaticParams> m_static_params;
     size_t m_N = 0;
     size_t m_K = 0;
+    size_t m_stride = 0;
```

Comment:
To be honest, it's unclear from the naming what is this stride. Can we rename it? For example, to `copy_B_wei_stride` to align with [x64](https://github.com/openvinotoolkit/openvino/blob/c3f010c3103ffee12fd87fb2cdbe2b647cedc1c6/src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm_copy_b.hpp#L165) or `input_stride`?